webcrawler.py

```diff
@@ -5,12 +5,21 @@
 import scrapy
 from scrapy.crawler import CrawlerProcess
 from scrapy.http.request import Request
+import re
+
 
 KEYWORDS_FILE = 'textfile.txt'
 SCRAPY_OUTPUT_FILE = 'scrapy-output.json'
 RANKED_OUTPUT_FILE = 'output.csv'
 
 
+def contains_date(url):
+    result= re.findall(r'/(\d{4})/(\d{1,2})/(\d{1,2})/', url)
```

Comment:
Read the comment below on `if` first. `re.findall` is unnecessary, since you don't actually care about finding all instances of this pattern, but just the one. This means you should use [`re.search`](https://docs.python.org/dev/library/re.html#re.search) instead.

Also, note that the match objects returned by `re.search` is truthy...